include/pdal/Metadata.hpp

```diff
@@ -560,7 +560,7 @@ class PDAL_DLL MetadataNode
     }
 
     bool hasChildren() const
-        { return m_impl->m_subnodes.size(); }
+        { return (m_impl->m_subnodes.size()) != 0; }
```

Comment:
Gah.  I hate this so much.  Why must we state the obvious?  Can we suppress whatever warning this is?